### PR TITLE
Integrate upstream to Ansible pipelines

### PR DESCRIPTION
**Pull request by J0zi**
Hi CVP team.
All changes are for upstream only and should not affect original pipeline.

docker install
#quay login
docker install
KIND install
KIND start
kubectl installed
replaced oc with kubectl for upstream
fixed api version for operatorgroup - cannot work without this fix for both upstream and Openshift
create namespace
start catalog
added functionality with waiting on operator to stay alive 1 minute
original parser enabled for upstream
labels added
added info about git clone for catalog

**Commands:**
ansible-playbook -vv -i <machine>,  local.yml -e run_prereqs=false -e run_upstream=true --tags install,catalog_build

ansible-playbook -vv -i <machine>,   local-test-operator.yml  \
-e run_upstream=true -e run_prereqs=false -e run_imagesource=false \
-e operator_dir="/tmp/community-operators-for-catalog/upstream-community-operators/aqua"

### CODE REVIEW

**Comment by nmars** [comment]
The failed check for **CVP/cvp-isv-operator-metadata-validation-test** was due to this failure when running operator-scorecard-test.yml...

`06:05:36  TASK [operator_scorecard_tests : Delete the operator CR] ***********************
06:05:36  task path: /home/jenkins/agent/workspace/cvp-isv-operator-metadata-validation-test/operators/config/ansible/roles/operator_scorecard_tests/tasks/main.yml:34
06:05:36  fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'run_upstream' is undefined\n\nThe error appears to be in '/home/jenkins/agent/workspace/cvp-isv-operator-metadata-validation-test/operators/config/ansible/roles/operator_scorecard_tests/tasks/main.yml': line 34, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: \"Delete the operator CR\"\n  ^ here\n`

**Comment by J0zi** [comment]
@nmars, please have a look, adjusted:)

File: deploy-olm-operator.yml [hunk 1]
```diff
@@ -8,21 +8,26 @@
...
+    run_upstream: false
     openshift_namespace: "test-operator"
     operator_work_dir: "/home/jenkins/agent/test-operator"
     scorecard_cr_dir: "/home/jenkins/agent/scorecard-cr-files"
     work_dir: "{{ lookup('env', 'WORKSPACE') }}"
     olm_operator_files_path: "{{ work_dir }}/olm-operator-files"
     quay_token: "{{ cvp_vault.quay_token }}"
     quay_namespace: "cvpops"
-    oc_bin_path: "oc"
+    run_upstream: false
```
Comment (nmars) [inline deploy-olm-operator.yml:19]: Nit: duplicates line 11

**Comment by J0zi** [comment]
@nmars, fixed in #27. Closing, please continue review in #27. Thank you very much.

**Pull request closed without merging**